src/librados/librados_cxx.cc

```diff
@@ -1881,6 +1881,7 @@ static void rados_aio_getxattr_completepp(rados_completion_t c, void *arg) {
     rc = cdata->bl->length();
   }
   cdata->completion.finish(rc);
+  ((librados::AioCompletionImpl*)c)->put();
```

Comment:
can we use the cast operator like
```c++
reinterpret_cast<AioCompletionImpl*>(c)->put();
```
please note, we are already in the namespace of `librados` here.